### PR TITLE
Feat/api/get(User|engagement)DetailsById

### PR DESCRIPTION
**Pull request by rishusingh022**

### CODE REVIEW

File: src/utils/makeRequest/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import axios from 'axios';
 import { BACKEND_URL } from '../../constants/apiEndpoints';
 import AUTHORIZATION_TOKEN from '../../constants/authorizationToken';
+//import AUTHORIZATION_TOKEN from '../../constants/authorizationToken';
```
Comment (rohini-senthil) [inline src/utils/makeRequest/index.js:4]: Let's remove this comment

File: src/pages/EngagementDetailsPage/index.jsx [hunk 2]
```diff
@@ -1,6 +1,21 @@
 import * as React from 'react';
 import './EngagementDetailsPage.css';
+
+import { useEffect, useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { GET_ENGAGEMENT_DATA_BY_ID_URL } from '../../constants/apiEndpoints';
+import { default as makeRequest } from '../../utils/makeRequest/';
+
 const EngagementDetailsPage = () => {
+  const { projectId } = useParams();
```
Comment (rohini-senthil) [inline src/pages/EngagementDetailsPage/index.jsx:10]: Can we add tests for that pages?

File: src/utils/makeRequest/index.js (hunk 1 above)
Comment (rishusingh022) [reply src/utils/makeRequest/index.js:4]: sure !

File: src/pages/PeopleDetailsPage/index.jsx [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+import React, { useEffect, useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { GET_USER_DATA_BY_ID_URL } from '../../constants/apiEndpoints';
+import { default as makeRequest } from '../../utils/makeRequest/';
+
 const PeopleDetailsPage = () => {
+  const { userId } = useParams();
+  const [userDetails, setUserDetails] = useState({});
+  const navigate = useNavigate();
+  useEffect(() => {
```
Comment (Chetan-MckinseyBEBR) [inline src/pages/PeopleDetailsPage/index.jsx:12]: Add empty lines between code sections appropriately

Comment (rishusingh022) [reply src/pages/PeopleDetailsPage/index.jsx:12]: okay sure !!

**Review by Chetan-MckinseyBEBR — APPROVED** [review]

**Review by rohini-senthil — APPROVED** [review]

**Pull request merged**